biochatter/database_agent.py

```diff
@@ -107,3 +111,24 @@ def get_query_results(self, query: str, k: int = 3) -> list[Document]:
                 break
 
         return documents
+
+    def get_description(self):
+        try:
+            result = self.driver.query("MATCH (n:Schema_info) RETURN n LIMIT 1")
+
+            if result[0]:
+                schema_info_node = result[0][0]["n"]
+                MAX_SCHEMA_INFO_LENGTH = 1000
+                schema_dict_content = schema_info_node["schema_info"][:MAX_SCHEMA_INFO_LENGTH] # limit to 1000 characters
+                return (f"the graph database contains the following nodes and edges: \n\n"
+                        f"{schema_dict_content}")
+        except Exception:
```

Comment:
@fengsh27 can we be more specific here? what kind of exception do we expect? do we need to inform the user this happened (logging, printing, raising)?

Reply:
it's not appropriate to use try-catch here